tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -195,6 +210,19 @@ def make_response(cls, score, include_score, include_status, status_code):
             }
         )
 
+    @classmethod
+    def make_malformed_response(cls):
+        return xmltodict.unparse(
+            {
+                # "imsx_POXEnvelopeResponse" is the expected key
+                "imsx_POXEnvelopeRequest": {
```

Comment:
Does not much matter what this value is as long as its not imsx_POXEnvelopeResponse, but this is actually what blackbaud returns (incorrectly), so I figured I would just match its brokenness

Reply:
Do you have an example of a complete response from Blackbaud? Does it always return an unexpected response structure or only in certain scenarios?